HW0/P6/P6B.py

```diff
@@ -15,14 +16,30 @@ def burnTime(t):
     pool = mp.Pool(P)
 
     # Use a variety of wait times
-    ratio = []
-    wait_time = []
-
+    ratio =np.zeros(7)
```

Comment:
Yes, I obtained this same graph when running my code in the virtual machine. When I ran it in the Ipython notebook I got what everybody else has. I have no idea why....
I ran your code in an ipython notebook and the graph is ok :+1:  , I attach it here so you can see.
![real_graph](https://cloud.githubusercontent.com/assets/8644361/10006785/2babab16-608e-11e5-9694-465174f4c98d.png)